packages/integration-test/src/config.js

```diff
@@ -0,0 +1,68 @@
+const BN = require('bn.js');
+const {
+    constants: { ERC20_SCALING_FACTOR },
+} = require('@aztec/dev-utils');
+
+/**
+ * @module config
+ *
+ * config file to setup the integration test according to the specific scenario and environment the
+ * user wishes to test. Parameters that can be customised according to the test include:
+ * - testNet to run the integration test on
+ * - contracts to be tested
+ * - number of tokens that are involved in the full test suite, and therefore should be minted
+ * - flags controlling whether particular tests, such as minting, burning and upgrade paths, should be run
+ */
+const config = {};
+
+/**
+ * @param {String[]} contractsToDeploy - names of the various contracts for which Truffle contracts,
+ * representing those contracts deployed on the relevant testNet, should be created. These Truffle
+ * contracts will then be used for the relevant integration test
+ */
+config.contractsToDeploy = [
+    'ACE',
+    'Dividend',
+    'ERC20Mintable',
+    'FactoryAdjustable201907',
+    'JoinSplit',
+    'JoinSplitFluid',
+    'PrivateRange',
+    'PublicRange',
+    'Swap',
+    'ZkAssetAdjustable',
+];
+
+/**
+ * @constant {string} NETWORK - name of the test network for which the integration test is to be run.
+ * Extracted from the flag passed to the truffle test command - it is the last flag to be passed
```

Comment:
```suggestion
 * Extracted from the arguments passed to the truffle test command - it is the last argument to be passed
```
This is really pedantic, but flags in unix != arguments

Reply:
Good point, done